nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-security/src/test/groovy/org/apache/nifi/framework/security/util/SslContextFactoryTest.groovy

```diff
@@ -0,0 +1,172 @@
+        assertNull Security.getProperty("ocsp.enable")
+    }
+
+    @Test
+    void testCreateSslContextOCSPEnabled() {
+
+        // Arrange
+        Properties nifiProps = new Properties();
+        nifiProps.setProperty(NiFiProperties.SECURITY_OCSP_ENABLED, "true")
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE, ksFile.getAbsolutePath())
+        nifiProps.setProperty(NiFiProperties.WEB_HTTP_HOST, "localhost")
+        nifiProps.setProperty(NiFiProperties.WEB_HTTPS_HOST, "secure.host.com")
+        nifiProps.setProperty(NiFiProperties.WEB_THREADS, NiFiProperties.DEFAULT_WEB_THREADS.toString())
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE, ksFile.getAbsolutePath())
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE_TYPE, "JKS")
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE_PASSWD, keystorePass)
+        nifiProps.setProperty(NiFiProperties.SECURITY_TRUSTSTORE, trustFile.getAbsolutePath())
+        nifiProps.setProperty(NiFiProperties.SECURITY_TRUSTSTORE_TYPE, "JKS")
+        nifiProps.setProperty(NiFiProperties.SECURITY_TRUSTSTORE_PASSWD, "passwordpassword")
+        NiFiProperties mockProps = new StandardNiFiProperties(nifiProps)
+
+        // Act
+        SSLContext fac = SslContextFactory.createSslContext(mockProps)
+
+        // Assert
+        assert fac
+        assert fac.contextSpi.properties.get("x509TrustManager").pkixParams.revocationEnabled
+        assert Security.getProperty("ocsp.enable")
+    }
+
+    @Test
+    void testCreateSslContextOCSPDisabled() {
+
+        // Arrange
+        Properties nifiProps = new Properties()
+        nifiProps.setProperty(NiFiProperties.SECURITY_OCSP_ENABLED, "false")
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE, ksFile.getAbsolutePath())
+        nifiProps.setProperty(NiFiProperties.WEB_HTTP_HOST, "localhost")
+        nifiProps.setProperty(NiFiProperties.WEB_HTTPS_HOST, "secure.host.com")
+        nifiProps.setProperty(NiFiProperties.WEB_THREADS, NiFiProperties.DEFAULT_WEB_THREADS.toString())
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE, ksFile.getAbsolutePath())
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE_TYPE, "JKS")
+        nifiProps.setProperty(NiFiProperties.SECURITY_KEYSTORE_PASSWD, keystorePass)
+        nifiProps.setProperty(NiFiProperties.SECURITY_TRUSTSTORE, trustFile.getAbsolutePath())
+        nifiProps.setProperty(NiFiProperties.SECURITY_TRUSTSTORE_TYPE, "JKS")
+        nifiProps.setProperty(NiFiProperties.SECURITY_TRUSTSTORE_PASSWD, "passwordpassword")
+        NiFiProperties mockProps = new StandardNiFiProperties(nifiProps)
+
+        // Act
+        SSLContext fac = SslContextFactory.createSslContext(mockProps)
+
+        // Assert
+        assert fac
+        assertNull fac.contextSpi.properties.get("x509TrustManager").pkixParams
+        assertNull Security.getProperty("ocsp.enable")
+    }
+
```

Comment:
I don't think any of these unit tests exercise the `ocsp.responderUrl` property.